res/values-zh-rCN/yaap_strings.xml

```diff
@@ -51,8 +51,8 @@
     <string name="update_settings_title">YAAP 更新</string>
     <string name="update_settings_summary">检查系统更新</string>
 
-    <!-- String for removal of sensitive info on about, depending on tap -->
-    <string name="device_info_protected_single_press">点击显示</string>
```

Comment:
Did you remove these intentionally ?

Reply:
i moved it to [here](https://github.com/yaap/packages_apps_Settings/pull/45/files#diff-d37a05c519520f1f24d434c0fcf7ab773ec8147effea4b67e17528ea27d6742fR380)

Reply:
OH! cool